install-panel.sh

```diff
@@ -136,35 +136,16 @@ required_input() {
 }
 
 password_input() {
-  local  __resultvar=$1
-  local  result=''
-
-  while [ -z "$result" ]; do
-    echo -n "* ${2}"
-
-    # modified from https://stackoverflow.com/a/22940001
-    while IFS= read -r -s -n1 char; do
-      [[ -z $char ]] && { printf '\n'; break; } # ENTER pressed; output \n and break.
-      if [[ $char == $'\x7f' ]]; then # backspace was pressed
-          # Only if variable is not empty
-          if [ -n "$result" ]; then
-            # Remove last char from output variable.
-            [[ -n $result ]] && result=${result%?}
-            # Erase '*' to the left.
-            printf '\b \b' 
-          fi
-      else
-        # Add typed char to output variable.
-        result+=$char
-        # Print '*' in its stead.
-        printf '*'
-      fi
+  local result=$1
+  echo -en "* ${2}\n"
+  while [[ -z "$password_main" || ! "$password_main" == "$password_confirm" ]]
+    do
+      read -rsp "Password: " password_main && echo
+      read -rsp "Confirm Password: " password_confirm && echo
```

Comment:
You resolved it [in my previous review](https://github.com/vilhelmprytz/pterodactyl-installer/pull/120#discussion_r538926268), but I still don't understand the use of `-p`. If you look at the code as it was before, it doesn't use `-p` yet it can accept user input.

Reply:
it's -p "Confirm Password: "
so it displays the Confirm Password